src/components/02_atoms/OpsModalButton/OpsModalButton.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { storiesOf } from '@storybook/react';
+
+import EditIcon from '@material-ui/icons/Edit';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import OpsModalButton from './OpsModalButton';
+
+storiesOf('Table Operations/OpsModalButton', module).addWithJSX(
```

Comment:
Couldn't this component be used outside of tables? 🤔

Reply:
> Couldn't this component be used outside of tables?

Maybe, 

The Ops in OpsModalButton - is short for 'object in the operations column of a table'.

I am still thinking of what are the next steps.... I could put the table as a decorator so when a user reads the bottom section of the OpsModalButton he/she is not immediately distracted.

Or maybe there would be two pages one showing the button in isolation and one with the button in the table? I will look at this some more and fix this up tonight.